src/main/java/bio/terra/cda/app/service/QueryService.java

```diff
@@ -54,24 +72,38 @@ private Job runJob(Job queryJob) {
     }
   }
 
-  public List<String> runQuery(Query query) {
-    String queryString = generateQueryClause(query);
+  private Integer defaultValue(Integer val, Integer def) {
```

Comment:
In general, only used boxed types (like Integer or Double) if the value can be `null`.
```suggestion
  private int defaultValue(Integer val, int def) {
```
Also there may be a method in `Objects` that does this.